Objects/Objects/Other/Instance.cs

```diff
@@ -39,7 +39,7 @@ protected virtual IEnumerable<Base> GetTransformableGeometry()
     var displayValueRule = TraversalRule
       .NewTraversalRule()
       .When(DefaultTraversal.HasDisplayValue)
-      .ContinueTraversing(_ => DefaultTraversal.displayValueAndElementsPropAliases);
+      .ContinueTraversing(_ => DefaultTraversal.GeometryPropAliases);
```

Comment:
Oh no! 
There's a mistake in my Obsolete tag for `displayValueAndElements`
The correct name is `DisplayValueAndElementsPropAliases.

![image](https://github.com/specklesystems/speckle-sharp/assets/45512892/e1e2b631-989b-4632-84e5-ba294cb93d9f)

Please can you correct this, and the `Obsolete` message in Core

Reply:
Yup will do!